sparta/src/Simulation.cpp

```diff
@@ -480,6 +480,12 @@ void Simulation::configure(const int argc,
                                                sim_config_->warnings_file));
     }
 
+    // FIXME: Support debug-roi for loggers
+    if(sim_config_->trigger_on_type == SimulationConfiguration::TriggerSource::TRIGGER_ON_ROI &&
+       !sim_config_->getTaps().empty()) {
+        throw SpartaException("debug-roi hasn't been supported for loggers");
```

Comment:
Can you change the error message to:
"Logging ennoblement is currently not supported with debug-roi.  Use --debug or --debug-on-icount"

Reply:
And done